workshops/filters.py

```diff
@@ -1,8 +1,12 @@
-import django.forms.widgets
+import re
 
+import django.forms.widgets
 import django_filters
+from django.db.models import Q
+from django.forms import widgets
```

Comment:
if you import `widgets` from `django.forms`, why import `django.forms.widgets` separately?